layout-management-webapps/src/main/webapp/vue-app/common-layout-components/components/site-navigation/NodeItemMenu.vue

```diff
@@ -322,12 +323,12 @@ export default {
         });
 
     },
-    undoDeleteNode(nodeId) {
+    undoDeleteNode(nodeId, successMsg) {
```

Comment:
is there a need to pass the message as a parameter ?

Reply:
the i18N message should be defined out of the callback function or it will throw an error and could not be read

Reply:
Ok, I see